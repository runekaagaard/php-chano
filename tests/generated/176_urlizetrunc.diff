001- <a href="http://www.google.co.uk/search?hl=en&q=some+long+url&btnG=Search&meta=" rel="nofollow">http://www.google...</a>
001+ Fatal error: Call to undefined method DtlIter::urlizetrunc() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/176_urlizetrunc.php on line 4