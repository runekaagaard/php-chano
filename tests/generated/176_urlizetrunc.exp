<a href="http://www.google.co.uk/search?hl=en&q=some+long+url&btnG=Search&meta=" rel="nofollow">http://www.google...</a>