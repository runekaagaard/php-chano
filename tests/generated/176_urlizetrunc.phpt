--TEST--
A generated testfile for the "urlizetrunc" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'http://www.google.co.uk/search?hl=en&q=some+long+url&btnG=Search&meta=')));
foreach ($items as $i) echo $i->input->urlizetrunc(20);
--EXPECT--
<a href="http://www.google.co.uk/search?hl=en&q=some+long+url&btnG=Search&meta=" rel="nofollow">http://www.google...</a>