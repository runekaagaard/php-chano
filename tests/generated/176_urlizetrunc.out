Fatal error: Call to undefined method DtlIter::urlizetrunc() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/176_urlizetrunc.php on line 4